routeid/routeid.go

```diff
@@ -0,0 +1,29 @@
+package routeid
```

Comment:
options:
- for something as trivial as this, even code duplication is fine
- it can be put into a package that already exists and used by both depending packages.

Please don't create a package for a single function, either above solution is better.